packages/sveltekit/src/server/serverRoute.ts

```diff
@@ -0,0 +1,66 @@
+import { SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN, SEMANTIC_ATTRIBUTE_SENTRY_SOURCE, startSpan } from '@sentry/node';
+import { addNonEnumerableProperty } from '@sentry/utils';
+import type { RequestEvent } from '@sveltejs/kit';
+import { flushIfServerless, sendErrorToSentry } from './utils';
+
+type PatchedServerRouteEvent = RequestEvent & { __sentry_wrapped__?: boolean };
+
+/**
+ * Wraps a server route handler for API or server routes registered in `+server.(js|js)` files.
+ *
+ * This function will automatically capture any errors that occur during the execution of the route handler
+ * and it will start a span for the duration of your route handler.
+ *
+ * @example
+ * ```js
+ * import { wrapServerRouteWithSentry } from '@sentry/sveltekit';
+ *
+ * const get = async event => {
+ *   return new Response(JSON.stringify({ message: 'hello world' }));
+ * }
+ *
+ * export const GET = wrapServerRouteWithSentry(get);
+ * ```
+ *
+ * @param originalRouteHandler your server route handler
+ * @param httpMethod the HTTP method of your route handler
+ *
+ * @returns a wrapped version of your server route handler
+ */
+export function wrapServerRouteWithSentry(
+  originalRouteHandler: (request: RequestEvent) => Promise<Response>,
+): (requestEvent: RequestEvent) => Promise<Response> {
+  return new Proxy(originalRouteHandler, {
+    apply: async (wrappingTarget, thisArg, args) => {
+      const event = args[0] as PatchedServerRouteEvent;
+      const routeId = event.route && event.route.id;
+      const httpMethod = event.request.method;
```

Comment:
ultra-nit: I'd move these to after the `event.__sentry_wrapped__` check so we don't run through them unnecessarily